octue/__init__.py

```diff
@@ -1,8 +1,15 @@
-import logging
+import os
 
-from .logging_handlers import LOG_FORMAT
+from .logging_handlers import FORMATTER_WITHOUT_TIMESTAMP, apply_log_handler
 from .runner import Runner
 
 
-__all__ = "LOG_FORMAT", "Runner"
-package_logger = logging.getLogger(__name__)
+__all__ = ("Runner",)
+
+
+if os.environ.get("PLATFORM") == "GOOGLE_CLOUD_RUN":
+    # Use a log handler with a formatter that doesn't include the timestamp in the log message context to avoid the date
+    # appearing twice in the Google Cloud Run logs (Google adds its own timestamp to log messages).
+    logger = apply_log_handler(__name__, formatter=FORMATTER_WITHOUT_TIMESTAMP)
+else:
+    apply_log_handler()
```

Comment:
why is this called without __name__? Was expecting a call to be similar other than the override of the handler...

Reply:
It doesn't make a difference whether it's given `__name__` or `None` so I'll give it `__name__` for symmetry

Reply:
Update: it matters a lot whether we use `__name__` or `None`/nothing for the argument to `apply_log_handler`. If we use `__name__`, the handler isn't applied to the root logger so we get every log message twice - once formatted (`__name__` logger) and once unformatted (root logger). Using `None` applies the handler to the root logger; all other loggers propagate messages up to it, so messages are only printed once and in the correct format.

@thclark